app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -665,6 +665,11 @@ def build_toolbar_hide_button(id)
       when "condition_remove"
         return true if x_active_tree == :condition_tree || !role_allows(:feature => "condition_delete")
       end
+    when "CustomButtonSet"
+      case id
+      when "ab_group_edit", "ab_group_delete", "ab_button_new"
```

Comment:
@jzigmund "ab_button_delete","ab_button_edit", "ab_button_simulate" buttons are missing here, please check for any others that might be missing.